data/json/items/gun/flintlock.json

```diff
@@ -90,5 +90,24 @@
     "built_in_mods": [ "bayonet" ],
     "valid_mod_locations": [ [ "accessories", 1 ], [ "underbarrel", 1 ] ],
     "flags": [ "NEVER_JAMS", "DURABLE_MELEE" ]
+  },
+  {
+    "id": "carbine_flintlock_double",
+    "copy-from": "rifle_flintlock",
+    "type": "GUN",
+    "name": "double-barrel carbine",
+    "description": "This is a compact muzzle-loading rifle combines a pair of rudimentary flintlock actions and two barrels.  Whereas an antique multiple-barrel flintlock is an intricate work of gunsmithing, this weapon is simpler yet still servicable.",
+    "valid_mod_locations": [ [ "accessories", 1 ] ],
+    "relative": {
+      "weight": 100,
+      "volume": -2,
+      "bashing": -2,
+      "range": -3,
+      "ranged_damage": -5,
+      "dispersion": 75,
+      "recoil": 20,
+      "reload": -300
```

Comment:
In general prefer `proportional` over `relative`. It helps your item stay in balance if the base item changes. `reload` and `dispersion` would both be good candidates here